src/index.js

```diff
@@ -1,13 +1,78 @@
 import todoApp from './modules/todoApp.js';
-import { displayItems } from './modules/element.js';
 import Task from './modules/task.js';
 import Store from './modules/localStorage.js';
+import addCheckboxEvent from './modules/statusUpdate.js';
+import displayItems from './modules/displayTask.js';
 import './style.css';
 
 // event
-// Load TodoList once page loads
-document.addEventListener('DOMContentLoaded', displayItems);
+// Display TodoList
+displayItems();
 
+// Edit and Delete task event
+const editEl = (e) => {
+  const moreIcon = e.target;
+  const todoWrap = moreIcon.parentElement.parentElement;
+  const moreIconWrapper = moreIcon.parentElement;
+  const descWrapper = todoWrap.firstElementChild.lastElementChild;
+  const desc = descWrapper.querySelector('.description');
+
+  const deleteIcon = document.createElement('div');
+  deleteIcon.innerHTML = '<ion-icon class=\'ionic delete-icon\' name="trash-outline"></ion-icon>';
+  const inputEl = document.createElement('input');
+  inputEl.type = 'text';
+  inputEl.classList.add('edit-input');
+  inputEl.value = desc.textContent;
+
+  // hide description and moreIcon
+  desc.classList.toggle('none');
+  moreIcon.classList.toggle('none');
+  // Append delete and input element for edit
+  moreIconWrapper.appendChild(deleteIcon);
+  descWrapper.appendChild(inputEl);
+  todoWrap.style.backgroundColor = '#f6f6c8';
+  inputEl.focus(); // Initiate focus
+
+  const idLength = todoWrap.id.length;
+  const taskId = Number(todoWrap.id[idLength - 1]);
+
+  inputEl.addEventListener('change', (e) => {
+    const input = e.target;
+    desc.textContent = input.value;
+    desc.classList.toggle('none');
+    moreIcon.classList.toggle('none');
+    todoWrap.style.backgroundColor = '';
+    Store.editTaskLocalStore(taskId, input.value);
+    deleteIcon.remove();
+    input.remove();
+  });
+
+  deleteIcon.addEventListener('click', () => {
+    const listContainer = document.querySelector('.todo-list-holder');
+    todoApp.removeTask(taskId);
+    while (listContainer.firstChild) { // Remove all task in the list
+      listContainer.removeChild(listContainer.firstChild);
+    }
```

Comment:
- [ ] <b>REQUIRED:</b> Clear all completed should only remove `completed tasks` not everything. Kindly do that to comply with project requirements.